### ISSUE TITLE
not working on some pages that have code input

### ISSUE THREAD

**Issue by ghost** [godbout/kindaVim.blahblah#194]
App doesn't work on pages that have markdown input or code cells like Jupyter Notebooks, for instance https://www.kaggle.com/

**Comment by godbout** [comment]
something a little related: https://github.com/godbout/kindaVim.docs/issues/180
in the sense that rather than changing behavior by apps, your issue would require (to have a proper UX for the user) to differentiate by SITE. which would be of course crazy to do.

**Comment by pmentor** [comment]
Yeah, adding to my browser in the `Key Mapping` seems to be working, just that I can't see the highlighted VIM cursor in normal mode should I expect that? or is it something to fix it?

**Comment by godbout** [comment]
> Yeah, adding my browser in the `Key Mapping` seems to be working; just that I can't see the highlighted VIM cursor in normal mode. Should I expect that? Or is it something to fix it?

yeah, that's normal. kV has several "modes". the best one (Accessibility Strategy) requires apps/inputs to implement the macOS Accessibility properly. because to be able to do proper text calculation and manipulation, you need a lot of info. if the app/input doesn't give info, or wrong one (like with Jupyter), then kV needs to use another mode: the Keyboard Strategy one. basically it tries to translate a Vim motion in macOS key combinations. but in that case you can't have a block cursor, because kV doesn't know anything about the text and can't calculate anything related to it. the Keyboard Strategy is more like a fallback. the best mode is definitely the Accessibility one.

**Comment by godbout** [comment]
nothing to do here. closing.
